### DISCUSSION TITLE
Preprompt

### DISCUSSION THREAD

**Discussion by Lorg0n** [ogios/huggingchat-api#3] [Q&A] [answered]
Can you make it so that you can set a preprompt for a model?

**Comment by ogios** [comment]
you mean like seperate the input text into system prompt and user prompt?

**Comment by Lorg0n** [comment]
Yeah, system prompt

**Comment by ogios** [comment]
yeah, i've updated a version for that is [`0.0.2.3`](https://github.com/ogios/huggingchat-api/releases/tag/v23)
add prompt injection but i'm not sure if it actually works, you can see it here in [wiki](https://github.com/ogios/huggingchat-api/wiki/Prompt-Injection)

plus, i've just found some bugs some where else, may do another update soon

Update: Updated to v24.

**Comment by ogios** [comment, marked as answer]
we have another solution for this:
https://github.com/Soulter/hugging-chat-api/pull/93

edit: update to v25 to support this.
